src/storage-controller/src/lib.rs

```diff
@@ -1916,6 +1922,84 @@ where
         let id = self.introspection_ids[&type_];
         self.append_to_managed_collection(id, updates).await;
     }
+
+    /// With the CRDB based timestamp oracle, there is no longer write timestamp
+    /// fencing. As in, when a new Coordinator, `B`, starts up, there is nothing
+    /// that prevents an old Coordinator, `A`, from getting a new write
+    /// timestamp that is higher than `B`'s boot timestamp. Below is the
+    /// implications for all persist transaction scenarios, `on` means a
+    /// Coordinator turning the persist txn flag on, `off` means a Coordinator
+    /// turning the persist txn flag off.
+    /// - `off` -> `off`: The following series of events seem possible:
+    ///   1. `A` writes at `t_0`, s.t. `t_0` > `B`'s boot timestamp.
+    ///   2. `B` writes at `t_1`, s.t. `t_1` > `t_0`.
+    ///   3. `A` writes at `t_2`, s.t. `t_2` > `t_1`.
+    ///   4. etc.
```

Comment:
With the fencing described below, this would not be possible.

Reply:
plus, one of them will fail because a coord panics when it cannot apply a write (I think)